config/opal_mca.m4

```diff
@@ -123,11 +123,7 @@ AC_DEFUN([OPAL_MCA],[
     # in the form DIRECT_[type]=[component]
     #
     AC_MSG_CHECKING([which components should be direct-linked into the library])
-    if test "$enable_mca_direct" = "yes" ; then
```

Comment:
Is there a reason to remove this test?  It catches the case where someone specifies `--enable-mca-direct` (with no value), right?

Reply:
That is already caught below. yes does not follow the format of framework-component so it will fail when we try to determine the framework component pair.